sgkit/stats/pedigree.py

```diff
@@ -0,0 +1,78 @@
+from typing import Hashable
+
+import numpy as np
+import xarray as xr
+from xarray import Dataset
+
+from sgkit import variables
+from sgkit.utils import conditional_merge_datasets, create_dataset
+
+
+def parent_indices(
+    ds: Dataset,
+    *,
+    sample_id: Hashable = variables.sample_id,
+    parent_id: Hashable = variables.parent_id,
```

Comment:
I think we should have a ``missing`` parameter which lets us specify the missing data character, and then throw an error if there's any IDs not in sample_id + missing.

How about ``missing="."``?

Reply:
This sounds reasonable to me.

Reply:
Good suggestion! Are others happy with `"."` as the default? I just chose it for the examples based upon similarity with the VCF spec.

Reply:
I think "." is as good as any. Another option would be "NA", but :shrug: